### PR TITLE
Feat: Model Refactor and Transactions List View

### PR DESCRIPTION
**Pull request by AlexZorkin**
- Refactored models to remove issuance and add separate models for Initiative Agreement and Admin Adjustment
- Squashed migrations
- Added a transactions database view which aggregates together transaction type entities
- Added pagination for transactions view
- Hooked up frontend table with filtering and sorting on transactions view

### CODE REVIEW

**Review by hamed-valiollahi — APPROVED** [review]
Nice work! I've just added one suggestion that might be worth considering.

File: backend/lcfs/db/migrations/versions/2024-02-23-15-55_df41556bd1ee.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+# revision identifiers, used by Alembic.
+revision = "df41556bd1ee"
+down_revision = "5d3a79582d21"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute("""
+    CREATE OR REPLACE VIEW transaction_view AS
```
[the remark this answers is not in the document]
Comment (AlexZorkin) [reply backend/lcfs/db/migrations/versions/2024-02-23-15-55_df41556bd1ee.py:20]: Looks great thank you. This approach would make the page load very quick.

[the remark this answers is not in the document]
Comment (hamed-valiollahi) [reply backend/lcfs/db/migrations/versions/2024-02-23-15-55_df41556bd1ee.py:20]: Happy to hear it speeds up the page load. Thank you, Alex!

**Comment by kevin-hashimoto** [comment]
@AlexZorkin doesnt look like there will be any big conflicts with #420

**Pull request merged**